CDTDatastoreTests/CDTQIndexCreatorTests.m

```diff
@@ -278,24 +278,6 @@
                 expect(name).to.beNil();
             });
 
-            it(@"doesn't support using the geo type", ^{
-                NSString *name =
-                    [im ensureIndexed:@[ @{ @"name" : @"asc" }, @{
-                        @"age" : @"desc"
-                    } ] withName:@"basic"
-                                 type:@"geo"];
-                expect(name).to.beNil();
-            });
-
-            it(@"doesn't support using the unplanned type", ^{
-                NSString *name =
-                    [im ensureIndexed:@[ @{ @"name" : @"asc" }, @{
-                        @"age" : @"desc"
-                    } ] withName:@"basic"
-                                 type:@"frog"];
-                expect(name).to.beNil();
-            });
```

Comment:
I thought you said you'd get an exception now -- can we test for that and keep these tests?